tests/data/cases/power_op_spacing.py

```diff
@@ -1,3 +1,8 @@
+m2 = None if not isinstance(dist, Normal) else μ** 2 + σ * 2
+m3 = None if not isinstance(dist, Normal) else μ**2 + σ * 2
+m4 = None if not isinstance(dist, Normal) else s**2 + σ * 2
```

Comment:
```suggestion
m2 = None if not isinstance(dist, Normal) else m** 2 + s * 2
m3 = None if not isinstance(dist, Normal) else m**2 + s * 2
m4 = None if not isinstance(dist, Normal) else s**2 + s * 2
```
Replace the Greek letters with Latin ones. If there is something unusual going on in a test, then readers will naturally think the test is about that unusual thing. In this case the Greek doesn't matter, so use less unusual characters.